homepage/site/homepageView.jsx

```diff
@@ -439,6 +439,80 @@ const Faq = () => {
           </p>
         </details>
+        <details>
+          <summary>
+            <div class="faq-section-header">
+              Why does Firefox Voice ask for all these permissions?
+            </div>
+          </summary>
+          <p>
+            Because Firefox Voice implements a wide variety of commands, it
+            needs permissions to do many things on your behalf.
+          </p>
+          <dl>
+            <dt>Access your data for all websites</dt>
+            <dd>
+              This permission allows Firefox Voice to inject functionality onto
+              any site. This is needed for scrolling, copying and pasting,
+              finding tabs, and many more features. This permission would make
+              it <em>possible</em> for Firefox Voice to extract information from
+              websites, but it does not do so. The Privacy Policy specifies what
+              information we collect (with your permission at installation time,
+              or as specified in the preferences). It does not include any
+              browsing or site information.
+            </dd>
+            <dt>Read and modify bookmarks</dt>
+            <dd>
+              This is used for commands that open, create, and delete bookmarks.
+            </dd>
+            <dt>Read and modify browser settings</dt>
+            <dd>
+              This is used to open your preferred homepage; your homepage
+              location is a browser setting.
+            </dd>
+            <dt>Get data from the clipboard</dt>
+            <dd>
+              This is used to "paste" a URL into newtab (which actually opens
+              the URL, and requires us to read the URL). Also in some cases we
+              read the clipboard, put new information into the clipboard, paste
+              it, then restore your previous clipboard content.
+            </dd>
+            <dt>Input data to the clipboard</dt>
+            <dd>
+              This is used for commands like "copy link" that copy data into the
+              clipboard.
+            </dd>
+            <dt>
+              Download files and read and modify the browser’s download history
+            </dt>
+            <dd>
+              This is used for commands like "save page" and "save screenshot"
+              that download files. The command "show downloads" requires
+              additional permissions that include download history.
+            </dd>
+            <dt>Read the text of all open tabs</dt>
+            <dd>
+              This permission is used for the command "find [query] in page",
+              which is the equivalent of ^F/⌘F.
+            </dd>
+            <dt>Access browsing history</dt>
+            <dd>
+              This is used to determine your preferred music service, and in the
+              future we plan to use it to detect other preferred services. E.g.,
+              if you visit `spotify.com` often/recently, then we presume that is
```

Comment:
I apparently forgot this isn't Markdown. I'll fix this along with what I'm guessing may be some incoming language changes.